datastore/tests/unit/test_client.py

```diff
@@ -934,6 +934,17 @@ def test_transaction_defaults(self):
             self.assertIs(xact, mock_klass.return_value)
             mock_klass.assert_called_once_with(client)
 
+    def test_read_only_transaction_defaults(self):
+        creds = _make_credentials()
+        client = self._make_one(credentials=creds)
+
+        patch = mock.patch(
+            'google.cloud.datastore.client.Transaction', spec=['__call__'])
```

Comment:
The `Transaction` constructor has no network requirements, so why do you mock it?